_data/sites.json

```diff
@@ -5787,6 +5787,24 @@
         ]
     },
 
+    {
+        "name": "GameStop",
+        "url": "https://www.bollyinside.com/articles/how-to-delete-a-gamestop-account/",
+        "difficulty": "hard",
+        "email": "GSContentRequests@gamestop.com",
+        "notes": "Account deletion requires contacting Customer Support.",
+        "notes_tr": "Hesabınızın silinmesi için müşteri hizmetleri ile iletişime geçmeniz gerekir.",
+        "notes_fr": "Pour supprimer votre compte contactez le support.",
+        "notes_it": "La cancellazione account richiede di contattare il servizio clienti.",
+        "notes_de": "Account-Löschung erfordert es den Kundensupport zu kontaktieren",
+        "notes_pt_br": "Para deletar a sua conta contate a assistência ao cliente",
+        "notes_cat": "Per esborrar el seu compte contacti amb l'assistència al client",
+        "notes_es": "Para borrar tu cuenta contacta con asistencia al cliente",
```

Comment:
Same comment as in other reviews, please provide translations only for languages you are fluent into.

Reply:
I got these translations from the official site.json file of this project. Should I remove them anyway?

Reply:
No need then, thanks.